hopsworks-api/src/main/java/io/hops/hopsworks/api/pythonDeps/PythonDepsService.java

```diff
@@ -588,6 +589,24 @@ public Response search(@Context SecurityContext sc,
         libsFound).build();
   }
 
+  
+  @GET
+  @Path("/diskusage")
```

Comment:
What's the purpose of this endpoint? 
If it is to give information to the users about the size of the project anaconda environment, it shouldn't do a `du` on the whole anaconda lib`, but on the project one. 
If the purpose is to give admin information about the size of the anaconda directories, the endpoint shouldn't be here, but in the admin section, and the AllowedRoled shouldn't be `DATA_OWNER` but `HOPS_ADMIN`

Reply:
It's for the admin to find out about the size of all of the anaconda libraries on each host. Will fix.

Reply:
This REST API isn't currently used. The mgmt bean is used. I can remove it.